Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs

```diff
@@ -851,6 +851,10 @@ static bool CanSetValue(FieldReference bpRef, INode node, IXmlLineInfo iXmlLineI
 				return false;
 
 			var bpTypeRef = bpRef.GetBindablePropertyType(iXmlLineInfo, module);
+			var implicitOperator = varValue.VariableType.GetImplicitOperatorTo(bpTypeRef, module);
+			if (implicitOperator != null)
```

Comment:
using `op_implicit` on `SetValue` allows that to work on ABP or BP without setter defined